docs/learning/training/markdown-guide.md

```diff
@@ -14,7 +14,7 @@
 
 ## What is Markdown?
 
-Markdown (md) is a markup language, similar to a .txt file, but with basic **in-text** formatiing
+Markdown (md) is a markup language, similar to a .txt file, but with basic **in-text** formating
```

Comment:
Nice catch

Reply:
Actually it should be _formatting_!! My correction wasn’t even spelled correctly…